### DISCUSSION TITLE
FormTabs 中的字段布局模式不能继承顶层FormLayout的属性

### DISCUSSION THREAD

**Discussion by Nokecy** [alibaba/formily#1653] [Q&A 2.x] [answered]
https://codesandbox.io/s/funny-mclean-cq8p1?file=/App.tsx 
FormTabs里面的需要用独立的FormLayout 重新包裹一下，请问设计是这样的吗？

**Comment by janryWang** [comment, marked as answer]
看文档，FormLayout默认是shallow下发

**Reply by Nokecy** [reply]
~ 收到，明白了 谢谢
